lms/djangoapps/verify_student/views.py

```diff
@@ -1047,3 +1055,187 @@ def reverification_window_expired(_request):
+            'platform_name': settings.PLATFORM_NAME,
+            'purchase_endpoint': get_purchase_endpoint(),
+            'requirements': requirements,
+            'user_full_name': request.user.profile.name,
+            'verification_deadline': "",
+            'submit_photos_url': reverse('verify_student_incourse_reverification_confirmation')
+
+        }
+        return render_to_response("verify_student/incourse_reverify.html", context)
+
+    @method_decorator(login_required)
+    def post(self, request, course_id, checkpoint_name):
+        """
+        submits the reverification to SoftwareSecure
+        """
+        try:
+            user = request.user
+            course_id = CourseKey.from_string(course_id)
+            checkpoint = VerificationCheckpoint.get_verification_checkpoint(course_id, checkpoint_name)
+            if checkpoint is None:
+                raise VerificationCheckpointException
+            init_verification = SoftwareSecurePhotoVerification.get_initial_verification(user)
+            if not init_verification:
+                raise SoftwareSecurePhotoVerification.DoesNotExist
+            b64_face_image = request.POST['face_image'].split(",")[1]
+            attempt = SoftwareSecurePhotoVerification(user=request.user)
+            attempt.upload_face_image(b64_face_image.decode('base64'))
+            attempt.fetch_photo_id_image()
+            attempt.mark_ready()
+
+            attempt.save()
+            attempt.submit()
+
+            checkpoint.add_verification_attempt(attempt)
+
+            VerificationStatus.add_verification_status(checkpoint, user, "submitted")
+
+            return HttpResponseRedirect(reverse('verify_student_incourse_reverification_confirmation'))
+
+        except VerificationCheckpointException:
+            log.exception(
+                "Could not submit verification attempt for user {}".format(request.user.id)
+            )
+            context = {
+                "user_full_name": request.user.profile.name,
+                "error": True,
+            }
+            return render_to_response("verify_student/midcourse_photo_reverification.html", context)
+        except SoftwareSecurePhotoVerification.DoesNotExist:
+            log.exception(
+                "Could not submit verification attempt for user {}".format(request.user.id)
+            )
+            context = {
+                "user_full_name": request.user.profile.name,
+                "error": True,
+            }
+            return render_to_response("verify_student/midcourse_photo_reverification.html", context)
+        except Exception:
+            log.exception(
+                "Could not submit verification attempt for user {}".format(request.user.id)
+            )
+            context = {
+                "user_full_name": request.user.profile.name,
+                "error": True,
+            }
+            return render_to_response("verify_student/midcourse_photo_reverification.html", context)
```

Comment:
These responses seem to be rendering with the old "midcourse reverification" template.  Shouldn't these be using "incourse_reverify.html"?